modules/gapi/CMakeLists.txt

```diff
@@ -178,6 +181,15 @@ ocv_module_include_directories("${CMAKE_CURRENT_LIST_DIR}/src")
 ocv_create_module()
 
 ocv_target_link_libraries(${the_module} PRIVATE ade)
+
+if(CV_TRACE AND HAVE_ITT)
+  add_definitions(-DOPENCV_WITH_ITT=1)
+endif()
+if(ITT_INCLUDE_DIRS)
```

Comment:
can the above `if` fail and this one be `true`? Or vice-versa?

Reply:
I think, that it can't, because `HAVE_ITT` is  set to `1`  together with `ITT_INCLUDE_DIRS` is set to meaningful value when OpenCV CMakeLists.txt script is calling OpenCVDetectTrace.cmake script.

https://github.com/opencv/opencv/blob/master/CMakeLists.txt#L873
OpenCV CMakeLists.txt script is calling OpenCVDetectTrace.cmake script if `CV_TRACE` is `true`, what we also consider